src/Kunstmaan/NodeBundle/Resources/views/NodeAdmin/Modals/_pagetemplate.html.twig

```diff
@@ -0,0 +1,34 @@
+{% import _self as macros %}
+
+{% for row in pageTemplate.rows %}
+<div class="row">
```

Comment:
Yes indeed, when i format the twig file the output will contain all the spaces of the indentation. But that's not how we want the alignment, it should be like the real html structure.